CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+
+## v0.0.2
```

Comment:
Technically, when writing a changelog, the top most version should be something like **unreleased** and adjusted to the desired version on publish. This avoids conflicts when changes come from multiple PRs.

Reply:
Thanks for the Advise. I would say we should add that next Time.